assets/blocks/quiz/ordering-promo/index.js

```diff
@@ -0,0 +1,52 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { addFilter } from '@wordpress/hooks';
+import { ExternalLink } from '@wordpress/components';
+
+const PROMO_LINK =
+	'https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=quiz_ordering_question_type';
+
+function addOrderingPromoOption( options ) {
+	options.push( {
+		title: __( 'Ordering', 'sensei-lms' ),
+		description: __(
+			'Place the answers in the correct order.',
+			'sensei-lms'
+		),
+		label: __( 'Ordering', 'sensei-lms' ),
+		value: 'ordering',
+		disabled: true,
+	} );
+	return options;
+}
+
+addFilter(
+	'senseiQuestionTypeToolbarOptions',
+	'sensei-lms/ordering-promo',
+	addOrderingPromoOption
+);
+
+function addPromoLink( children, option ) {
+	if ( option.value !== 'ordering' ) {
+		return children;
+	}
+	return (
+		<div className="sensei-lms-question-block__type-selector__option__container--disabled">
+			<strong> { option.title }</strong>
+			<div className="sensei-lms-question-block__type-selector__option__description--disabled">
+				{ option.description }
+			</div>
+			<ExternalLink href={ PROMO_LINK }>
+				{ __( 'Upgrade to Sensei Pro', 'sensei-lms' ) }
+			</ExternalLink>
```

Comment:
Maybe add a bit of margin between the link and the description? I've noticed that the other descriptions have a `margin-top: 3px`.

Reply:
Fixed here: https://github.com/Automattic/sensei/pull/4815/commits/583459c71bf38519567c1e27f23e2c3f1f1da051